records_mover/db/redshift/loader.py

```diff
@@ -23,11 +23,20 @@ class RedshiftLoader(LoaderFromRecordsDirectory):
     def __init__(self,
                  db: Union[sqlalchemy.engine.Engine, sqlalchemy.engine.Connection],
                  meta: sqlalchemy.MetaData,
-                 temporary_s3_directory_loc: Callable[[], ContextManager[BaseDirectoryUrl]])\
+                 s3_temp_base_loc: Optional[BaseDirectoryUrl])\
             -> None:
         self.db = db
         self.meta = meta
-        self.temporary_s3_directory_loc = temporary_s3_directory_loc
+        self.s3_temp_base_loc = s3_temp_base_loc
+
+    # TODO: Do I want to combine with one below?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker